app/models/mixins/ui_service_mixin.rb

```diff
@@ -1,21 +1,24 @@
 module UiServiceMixin
+  # rubocop:disable Metrics/LineLength
```

Comment:
From a "historical" point of view, Rubocop was not always enabled, hence there are a lot of things in older source code that look like something that Rubocop won't approve.
But it is ran now on the newer code to ensure that at least the new code is more compliant with it.